### PR TITLE
Fix misc issues

### PR DESCRIPTION
**Pull request by ShellyXueHan**
1. styling
2. content update
3. redirect with clean state

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
-  border-style: solid;
-  border-width: 1px;
-  border-color: lightgray;
 }
 
 .container * {
   box-sizing: border-box;
 }
 
+.control-lable {
```
Comment (patricksimonian) [inline src/components/App.css]: `.control-label`

File: src/components/App.css [hunk 2]
```diff
@@ -156,3 +154,7 @@ input.form-control :focus {
 div {
   text-align: center;
 }
+
+h1 {
+  margin-bottom: 50px;
+}
```
Comment (patricksimonian) [inline src/components/App.css:63]: new line

File: src/containers/Confirmation.js [hunk 3]
```diff
@@ -41,24 +45,38 @@ class Confirmation extends Component {
     const emailJwt = localStorage.getItem('emailJwt');
     // TODO: maybe redirect after?
     const verifiedContent = this.props.confirmed ? (
-      <h4>Confirmed!</h4>
+      <h5>Confirmed!</h5>
     ) : (
       <div>
-        <h4>Do you want to confirm registration for email: {this.props.email} ?</h4>
-        <button
+        <br />
```
Comment (patricksimonian) [inline src/containers/Confirmation.js]: brs should be used as spacers between tags. Only spacer between text.

File: src/containers/Confirmation.js [hunk 4]
```diff
@@ -41,24 +45,38 @@ class Confirmation extends Component {
...
+        <h5>Do you want to confirm registration for email: </h5>
+        <h5>{this.props.email}</h5>
+        <br />
```
Comment (patricksimonian) [inline src/containers/Confirmation.js]: same same

File: src/components/App.css (hunk 2 above)
Comment (ShellyXueHan) [reply src/components/App.css:63]: removed unused css

**Comment by ShellyXueHan** [comment]
@patricksimonian fixed issues. need the style now so going to merge...

**Pull request merged**